src/Entity/ContentMagicTraits.php

```diff
@@ -55,6 +55,17 @@ public function get(string $name, array $arguments = [])
         }
     }
 
+    public function has(string $name): bool
+    {
+        foreach ($this->fields as $field) {
+            if ($field->getName() === $name) {
```

Comment:
if we could make $this->fields a name-indexed array, using this method be much faster (simple isset instead of foreaching).